src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    public static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    public static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "804148DimaDima");
+    }
```

Comment:
Storing database URL, username, and password directly in the code is a security risk. Consider using environment variables or a configuration file that is not included in the version control system. This way, you can keep sensitive information out of the codebase and reduce the risk of exposing it.